spec/src/main/asciidoc/microprofile-opentracing.asciidoc

```diff
@@ -192,7 +192,7 @@ server endpoint.
 The skip pattern is specified as a string with key `mp.opentracing.server.skip-pattern` which has to be
 compliant with `java.util.regex.Pattern`. An example skip pattern might be `mp.opentracing.server.skip-pattern=/health|/metrics.*`
 
-The default value of the skip pattern excludes tracing for default endpoints defined by MicroProfile Health, Metrics and OpenAPI specification.
+The default value of the skip pattern excludes tracing for default endpoints defined by MicroProfile Health, Metrics and OpenAPI specification. Defining a skip pattern does not override the default value.  Instead, both skip pattern and the default value are combined.
```

Comment:
I think we should rephrase it completely and instead just say that endpoits exposed by Metrics, Health.. are not traced.

Reply:
@pavolloffay Revised the wording.  Please review again. Thanks!